pkg/codec/codec.go

```diff
@@ -8,14 +8,25 @@ import (
 	"github.com/pion/mediadevices/pkg/prop"
 )
 
+// Name represents codec official name. It's possible to have more than 1 implementations
+// for the same codec name, e.g. openh264 vs x264.
+type Name string
+
+const (
+	NameOpus = "opus"
+	NameH264 = "H264"
+	NameVP8  = "VP8"
+	NameVP9  = "VP9"
```

Comment:
```suggestion
	NameOpus Name = "opus"
	NameH264 Name = "H264"
	NameVP8  Name = "VP8"
	NameVP9  Name = "VP9"
```
would be more explicit.

Reply:
Ouch, I forgot...